### DISCUSSION TITLE
Getting build error after deploying the dice game challenges to vercel in Speedrunethereum

### DISCUSSION THREAD

**Discussion by Nobleraj** [Cyfrin/foundry-full-course-cu#2292] [General]
Hey Team,

I'm solving some challenges in the Speedrunethereum, currently solving Dice game challenge.

I'm able to complete the challenge, test case are running local and did all the stuff's as mentioned in the challenge and deployed the contract in sepolia network.

But in the speedrunethereum website, for the dice game challenge i'm getting the below error for the rejection.

```
TypeError: Member "generateRandomNum" not found or not visible after argument-dependent lookup in contract DiceGame.
  --> contracts/download-0xac3532419d7ff4876f345ad035fc7802eaf5b385.sol:28:18:
   |
28 | 		uint256 roll = diceGame.generateRandomNum();
```

can someone guide me, what's going wrong here. i can do all the stuff's in local and deployed the contract in testnet also but i'm seeing this error .

Note: generateRandomNum function is implemented as expected

<img width="890" alt="Screenshot 2024-08-22 at 4 23 57 PM" src="https://github.com/user-attachments/assets/70870cf3-4357-4648-a21e-0a66ff873163">

**Comment by EngrPips** [comment]
Hello @Nobleraj, I hope you figure this out soon but the discussion tab of this Github is for discussing only the course content related to it. Please reach out to the `Speedrunethereum` team. Thanks

**Reply by Nobleraj** [reply]
Sure @EngrPips thanks

**Reply by EngrPips** [reply]
You are welcome, Friend.
